ddsp/training/data.py

```diff
@@ -53,20 +53,22 @@ def get_dataset(self, shuffle):
     """A method that returns a tf.data.Dataset."""
     raise NotImplementedError
 
-  def get_batch(self, batch_size, shuffle=True, repeats=-1):
+  def get_batch(self, batch_size, shuffle=True, repeats=-1,
+                drop_remainder=True):
```

Comment:
Please default to False so that you don't change the current default behavior (https://www.tensorflow.org/api_docs/python/tf/data/Dataset)

Reply:
My bad! Changed default value of drop_remainder in dataloader to False to match default behavior.